### ISSUE TITLE
Header hierarchy

### ISSUE THREAD

**Issue by grugnog** [CivicActions/nebula#84]
This seems like a fairly low priority accessibility issue (and a fair amount of effort to resolve) - the headers are not bad in that the h3 and h4 cover decent levels of content, however they are don't capture a complete hierarchy. Ideally they should look more this, however:
- h1. SideEffect.io
- h3 headers should be h2s
- There should be a "Sources and disclaimers" or similar h2 header to wrap that information
- All the source/disclaimer h4 headers should then be h3

If we are using HTML5 sections, then the header hierarchy should restart from h1 inside each section.

**Comment by rakanowicz** [comment]
Please see the style guide for size information:
[link](https://slack-files.com/files-pri-safe/T0297RSQF-F06RZMP43/sideeffect_styleguide.pdf?c=1435595578-41420fc7eabc8d11fb11fb801853f2acce364599)
